src/App.tsx

```diff
@@ -1,22 +1,18 @@
-import * as React from 'react';
-import './App.css';
-
-import logo from './logo.svg';
+import * as React from "react";
+import "./App.css";
+import intro from "./images/intro.png";
```

Comment:
I see that you have clipped the wireframe image and included it as-is. I think we want to create each of those small component separately and add them to our page. However, we can keep this as a placeholder for now if you want and look to break it into its component in another change. Let me know your thoughts on it.